membership-attribute-service/test/models/AnniversaryDateTest.scala

```diff
@@ -0,0 +1,22 @@
+package models
+
+import org.joda.time.LocalDate
+import org.specs2.mutable.Specification
+
+class AnniversaryDateTest extends Specification {
+  "anniversaryDate" should {
+    "if today is equal left then right" in {
```

Comment:
again left/right confused me a little, could they be closer to what those dates mean?